README.md

```diff
@@ -24,6 +24,183 @@ See [How_to_Install_a_Plugin](http://projects.theforeman.org/projects/foreman/wi
 | 1.18            | >= 1.5.0       |
 | 1.17            | >= 1.5.0       |
 
+## API Examples
+
+The api user requires following rights:
```

Comment:
That should be the same for the normal user.

Reply:
I would also add `... in Foreman` so you don't mix this up with the user foreman uses to connect to vSphere.